devdoc/singlylinkedlist2_requirements.md

```diff
@@ -0,0 +1,176 @@
+singlylinkedlist2 requirements
+================
+
+## Overview
+
+SinglyLinkedList is module that provides the functionality of a singly linked list, allowing its user to add, remove and iterate the list elements.
```

Comment:
>SinglyLinkedList [](http://example.com/codeflow?start=0&length=16)

Smells like slist. Or SList if you want to have it the same like DList.

I'd name the files slist.h and slist.md

Reply:
done